### PR TITLE
Added de_CH Provider

### PR DESCRIPTION
**Pull request by kstenschke**

### CODE REVIEW

File: src/Faker/Provider/de_CH/Person.php [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+        '{{firstNameFemale}} {{lastName}}',
+        '{{firstNameFemale}} {{lastName}}',
+        '{{firstNameFemale}} {{lastName}}',
+        '{{firstNameFemale}} {{lastName}}',
+        '{{titleFemale}} {{firstNameFemale}} {{lastName}}',
+        '{{firstNameFemale}} {{lastName}} {{suffix}}',
+        '{{titleFemale}} {{firstNameFemale}} {{lastName}} {{suffix}}',
+    );
+
+    protected static $firstNameMale = array(
```
Comment (fzaninotto) [inline src/Faker/Provider/de_CH/Person.php]: please quote your datasource in a PHP comment. Same for other lists.

**Comment by kstenschke** [comment]
Thank you fzaninotto for coming back to this pull request. please point me to an existing provider which does contain such a comment already, so i can see which conventions you'd expect.

**Comment by fzaninotto** [comment]
Somethink like https://github.com/fzaninotto/Faker/blob/f91f6a654c92ed2075d19b6142214d69cde06bab/src/Faker/Provider/en_GB/Person.php#L15-L18

**Comment by fzaninotto** [comment]
Abandoned PR, closing.

**Pull request closed without merging**